adapters/http-vertx-base/src/main/java/org/eclipse/hono/adapter/http/AbstractVertxBasedHttpProtocolAdapter.java

```diff
@@ -1020,6 +1029,9 @@ protected void setNonEmptyResponsePayload(final HttpServerResponse response, fin
                 }
 
             } else {
+                if (!requestProcessed.get()) {
+                    CommandConsumer.logReceivedCommandToSpan(command, waitForCommandSpan);
```

Comment:
The command is invalid in this case, isn't it? Shouldn't this result in an error being logged to the span?

Reply:
Yes, `logReceivedCommandToSpan` does  that.

Reply:
I see. Then FMPOV we should pull up the CommandConsumer.logReceivedCommandToSpan(command, waitForCommandSpan) invocation before the `if (isCommandValid(command, waitForCommandSpan))` statement and explicitly log the fact that the request has already been processed and the command is being released. WDYT?

Reply:
I've amended the commit accordingly.